intranet/apps/seniors/management/commands/cleanup_destinations.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
```

Comment:
You will need to run `./scripts/build_sources.sh` in your dev environment.

Reply:
Ok.